app/server/src/main/scala/org/bitcoins/server/Main.scala

```diff
@@ -193,6 +193,8 @@ object Main extends App with BitcoinSLogger {
     }
     startFut.failed.foreach { err =>
       logger.error(s"Error on server startup!", err)
+      err.printStackTrace()
```

Comment:
Can we add a unit test for this in some way? It seems that what we need?

Continuing discussion here: https://github.com/bitcoin-s/bitcoin-s/pull/1948#discussion_r481614347

Reply:
I'm not sure how we would, we can't really make tests for the server's `Main`

Reply:
`runMain()` is a method, you should be able to pass in bad configs somehow? We would have to make it non-private which i am ok with

Reply:
Make `runMain()` take in `args` so we can test ^